lib/tasks/statesman.rake

```diff
@@ -0,0 +1,73 @@
+namespace :statesman do
+  desc "Set most_recent to false for old transitions and to true for the "\
+       "latest one. Safe to re-run"
+  task :backfill_most_recent, [:parent_model_name] => :environment do |_, args|
+    parent_model_name = args.parent_model_name
+    abort("Parent model name must be specified") unless parent_model_name
+
+    parent_class = parent_model_name.constantize
+    transition_class = parent_class.transition_class
+    parent_fk = "#{parent_model_name.demodulize.underscore}_id"
+
+    total_models = parent_class.count
+    done_models = 0
+    batch_size = 500
+
+    parent_class.find_in_batches(batch_size: batch_size) do |models|
+      # Set historic transitions to have most_recent FALSE
+      #
+      # This selects transitions (initial_t) with a subsequent transition
+      # (subsequent_t), ensuring uniqueness by disallowing intermediate
+      # transitions (intermediate_t), and updates most_recent to be false.
```

Comment:
"The LEFT JOIN eliminates duplicate entries from the first join, for example, in:

initial_t.sort_key = 10, subsequent_t.sort_key = 20
initial_t.sort_key = 10, subsequent_t.sort_key = 30

it would eliminate the second row."